src/Checkbox.js

```diff
@@ -52,7 +51,7 @@ class Checkbox extends React.Component {
 
     if (inline) {
       const classes = {
-        [prefix(this.props, 'inline')]: true,
+        [prefix(props, 'inline')]: true,
```

Comment:
i'm slightly worried about this pattern, if someone refactors above to spread out bsClass this will subtly break.